resources/grafana-cfg.yaml

```diff
@@ -1,19 +0,0 @@
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: grafana-cfg
-  namespace: kube-system
-data:
-  grafana.ini: |
-    [server]
-    root_url = %(protocol)s://%(domain)s/web/grafana
-    [paths]
-    provisioning = /etc/grafana/provisioning
-    data = /var/lib/grafana
-    logs = /var/log/grafana
-    [auth.anonymous]
-    # enable anonymous access
-    enabled = true
-    [users]
-    # Default UI theme ("dark" or "light")
-    default_theme = dark
```

Comment:
If you remove this line, it won't be possible to change the Grafana dashboard theme by changing the ConfigMap